src/client/java/teammates/client/scripts/scalabilitytests/InstructorFeedbackResultsPageScalabilityTest.java

```diff
@@ -0,0 +1,70 @@
+package teammates.client.scripts.scalabilitytests;
+
+import org.testng.annotations.Test;
+
+import teammates.client.scripts.util.Stopwatch;
+import teammates.common.util.AppUrl;
+import teammates.common.util.Const;
+import teammates.common.util.Logger;
+import teammates.test.cases.browsertests.BaseUiTestCase;
+import teammates.test.pageobjects.InstructorFeedbackResultsPage;
+
+/**
+ * SUT: {@link Const.ActionURIs#INSTRUCTOR_FEEDBACK_RESULTS_PAGE}.
+ *
+ * <p>Run InstructorFeedbackResultsPageScaleTestDataGenerator.java before running the tests.
+ */
+public class InstructorFeedbackResultsPageScalabilityTest extends BaseUiTestCase {
+
+    private static final String DATA_FOLDER_PATH = "src/client/java/teammates/client/scripts/scalabilitytests/data/";
+    private static final Logger log = Logger.getLogger();
+
+    @Override
+    protected void prepareTestData() {
+        // the actual test data is refreshed before each test method
+    }
+
+    private void refreshTestData(int numStudents, int numQuestions) {
+        refreshTestData(
+                "InstructorFeedbackResultsPageScaleTest-" + numStudents + "Students" + numQuestions + "Questions.json");
+    }
+
+    private void refreshTestData(String filename) {
+        testData = loadDataBundle(DATA_FOLDER_PATH + filename);
+        removeAndRestoreDataBundle(testData);
+    }
+
+    @Test
+    public void testContentWithIncreasingLoad() throws Exception {
+        //Number of students and questions for each case.
+        int[] studentLoads = {10, 20};
+        int[] questionLoads = {1, 5, 10};
+
+        for (int studentLoad : studentLoads) {
+            for (int questionLoad : questionLoads) {
+                log.info("Testing with " + studentLoad + " students, " + questionLoad + " questions...");
+                refreshTestData(studentLoad, questionLoad);
+                loginToInstructorFeedbackResultsPage("CFResultsScT.instr", "Open Session");
+            }
+        }
+    }
+
+    @SuppressWarnings("PMD.UnnecessaryLocalBeforeReturn") // Needs to log before returning.
+    private InstructorFeedbackResultsPage loginToInstructorFeedbackResultsPage(String instructorName, String fsName) {
+        AppUrl resultsUrl =
+                createUrl(Const.ActionURIs.INSTRUCTOR_FEEDBACK_RESULTS_PAGE)
+                        .withUserId(testData.instructors.get(instructorName).googleId)
+                        .withCourseId(testData.feedbackSessions.get(fsName).getCourseId())
+                        .withSessionName(testData.feedbackSessions.get(fsName).getFeedbackSessionName());
+
+        Stopwatch stopwatch = new Stopwatch();
+        stopwatch.start();
+
+        InstructorFeedbackResultsPage resultsPage =
+                loginAdminToPage(resultsUrl, InstructorFeedbackResultsPage.class);
+
+        log.info("Time taken: " + stopwatch.getTimeElapsedInSeconds());
```

Comment:
Hang on, I misunderstood. @whipermr5 is right, the page load is not affected by the amount of responses anymore. The test is not meaningful unless it includes time taken for expanding at least one panel.